CONTRIBUTION_IDEAS.MD

```diff
@@ -0,0 +1,77 @@
+<!--
+******************************************************************************
+* 
+* Licensed under the Apache License, Version 2.0 (the "License");
+* you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at
+*
+*     http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*******************************************************************************/-->
+
+# oneTBB Contribution Ideas
+This file acts as a repository for potential contributions to the project.
+It provides a centralized location where contributors can find tasks and features to work on,
+helping to enhance the project. Its main purpose is to guide contributors towards areas where
+they can make a positive impact on the project's development.
+
+## Complexity Brackets
+Tasks in this repository are categorized into three complexity brackets: Beginner, Intermediate, and Expert.
+Each task is labeled with the appropriate complexity bracket to indicate the level of difficulty.
+
+- Beginner:These tasks are suitable for contributors who are new to the project or have limited experience.
+They involve relatively simple concepts and can be a good starting point for learning and getting familiar with the codebase.
+
+- Intermediate: These tasks require a moderate level of knowledge and experience.
+They may involve more complex features or require understanding of specific areas of the project.
+Contributors with some experience in the project can take on these tasks to further develop their skills.
+
+- Expert: These tasks are challenging and require advanced knowledge and expertise.
+They often involve complex algorithms, performance optimizations, or deep understanding of the project's internals.
+Contributors with extensive experience in the project can tackle these tasks to push the boundaries of the project's capabilities.
+
+When choosing a task to work on, consider your skill level and the complexity bracket that best matches your expertise.
+This will help ensure that you can make meaningful contributions and continue to grow as a developer.
+
+## Beginner
+
```

Comment:
Сonsider the following "beginner" task:
```suggestion
### Align symbol exporting rules across oneTBB libraries/plugins
oneTBB uses .def files to list global symbols to be exported and also local symbols to hide. 
For example, TBBMalloc hides symbols from static libraries which is a result of compilation with Intel Compiler (see https://github.com/oneapi-src/oneTBB/blob/master/cmake/compilers/Intel.cmake#L34). 
oneTBB library hides all symbols which are not listed in global section.
But other libraries like TBBBind, for example, for some reason does neither of these things and it may result in some symbol conflicts in them environment where multiple libraries depends on the same symbols.
The goal of this task is to align approaches across all of these libraries:
- Define local section with the symbols not meant for the exporting for each library.
- Analyze if is there any unintendedly exported symbols that are not related to the oneTBB libraries themselves. Consider hiding these symbols not in .def files but in the corresponding compiler CMake files.
```